### PR TITLE
Not stopping JWT active scanner on finding client side vulnerability

### PR DESCRIPTION
**Pull request by preetkaran20**

### CODE REVIEW

File: src/main/java/org/zaproxy/zap/extension/jwt/JWTActiveScanRule.java [hunk 1]
```diff
@@ -105,9 +105,7 @@ public void scan(HttpMessage msg, String param, String value) {
         }
 
         if (JWTConfiguration.getInstance().isEnableClientConfigurationScan()) {
-            if (performAttackClientSideConfigurations(msg, param)) {
-                return;
-            }
```
Comment (preetkaran20) [inline src/main/java/org/zaproxy/zap/extension/jwt/JWTActiveScanRule.java:110]: @kingthorin @thc202 i think there is no need to stop the addon on finding any UI related vulnerability.

Comment (thc202) [reply src/main/java/org/zaproxy/zap/extension/jwt/JWTActiveScanRule.java:110]: Makes sense.

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
DOH, I just realized you didn't include a CHANGELOG entry. Not that's it's a show stopper but probably worth mentioning for users since it potentially impacts active scanning and performance.

**Comment by preetkaran20** [comment]
> DOH, I just realized you didn't include a CHANGELOG entry. Not that's it's a show stopper but probably worth mentioning for users since it potentially impacts active scanning and performance.

ah missed that ... need to add some github action to make sure that change log is updated. sure will update it.

**Pull request merged**